fuzzers/007-timing/bel/tim2json.py

```diff
@@ -63,26 +93,59 @@ def pin_in_model(pin, model, direction=None):
             return True, pin
         elif extended_pin_name in model.split('_'):
             return True, extended_pin_name
+        elif (pin_aliases is not None) and (pin.upper() in pin_aliases):
+            for alias in pin_aliases[pin.upper()]['names']:
```

Comment:
Factor this loop into a function and add some tests

Reply:
done